### DISCUSSION TITLE
Forbidden (403)

### DISCUSSION THREAD

**Discussion by vitormicillo** [filamentphp/filament#1213] [General]
Hi guys, after install filament in as empty laravel project and followed all steps on docs, when I try to access my model I get error 403 Forbidden. 
Another problem found was that after running the resources command for my model it is not showing in the side menu.
Anyone here had the same problem?

Really appreciate for any help.

**Comment by danharrin** [comment]
Hey, do you have any policies defined for that model?

**Reply by vitormicillo** [reply]
Hi Dan, It's a new installation, I just followed the tutorial in the documentation.
The steps I followed:
- 1 After installing and configuring a new project, run migration to create the tables in the database.
- 2 install the filament and create the user
- 3 run artisan to create a model using the (-a) parameter so that it creates the model, controller, migration
- 4 I did the normal procedure to populate the database according to laravel standards
- 5 run php artisan make:filament-resource Code (Code is my Model name for table Codes)

What did I forget to do?

**Reply by vitormicillo** [reply]
Sorry mate, you're right, with the "-a" parameter it created the CodePolicy file and I hadn't seen it. 😆😆😆😆

**Reply by danharrin** [reply]
Yeah, you need to make sure that the policy allows you to `viewAny` the resource :)

**Reply by vitormicillo** [reply]
Really appreciate your help, I want to be the official maintainer in Italian language for the project.

**Reply by danharrin** [reply]
Awesome! That would be great. I think we already have Italian translations for most of the forms and tables packages, but are missing those for the admin panel.

**Comment by devsumitkarn** [comment]
how to create custom forbiden error
